sources/tests/Unit/Command/GenerateEntity.php

```diff
@@ -47,7 +47,7 @@ public function testExecute()
                 'schema'           => 'pomm_test',
                 'relation'         => 'alpha',
                 '--prefix-ns'      => 'Model',
-                '--prefix-dir'     => 'tmp',
+                '--prefix-dir'     => 'tmp'
```

Comment:
You are right, this wasn't intended as a fix. I probably had first thought of adding the new option at this point and forgot to restore the comma...